README.md

```diff
@@ -186,41 +188,42 @@ apiVersion: service.binding/v1alpha1
 kind: ServiceBinding
 metadata:
   name:                 # string
+  generation:           # int64, defined by the Kubernetes control plane
+  ...
 spec:
   name:                 # string, optional, default: .metadata.name
   type:                 # string, optional
   provider:             # string, optional
 
-  application:          # PodSpec-able resource ObjectReference-able
+  application:          # PodSpec-able resource ObjectReference-like
     apiVersion:         # string
     kind:               # string
     name:               # string, mutually exclusive with selector
     selector:           # metav1.LabelSelector, mutually exclusive with name
     containers:         # []intstr.IntOrString, optional
-    ...
 
-  service:              # Provisioned Service-able resource ObjectReference-able
+  service:              # Provisioned Service-able resource ObjectReference-like
     apiVersion:         # string
     kind:               # string
     name:               # string
-    ...
 
   mappings:             # []Mapping, optional
   - name:               # string
     value:              # string
-  ...
 
   env:                  # []EnvVar, optional
   - name:               # string
     key:                # string
-  ...
 
-status:
+status:                 # empty on create, populated by the reconciler
+  binding:              # LocalObjectReference, optional
+    name:               # string
   conditions:           # []Condition containing at least one entry for `Ready`
   - type:               # string
     status:             # string
     lastTransitionTime: # Time
     reason:             # string
+  observedGeneration:   # int64
     message:            # string
```

Comment:
this looks like a bad merge. message is part of the condition, not the observedGeneration

```suggestion
    message:            # string
  observedGeneration:   # int64
```